spec/csv_importer_spec.rb

```diff
@@ -633,8 +638,20 @@ class ImportUserCSVByFirstName
       end
 
       import.run!
+      expect(import.report.valid_rows.size).to eq(1)
```

Comment:
This is really confusing, you defined a rule to skip the import or not, that doesn't make input values invalid, just make this import not suitable to go in. I mean no need to find out what is wrong with the CSV.